xwayland/server.c

```diff
@@ -332,15 +345,24 @@ static bool server_start(struct wlr_xwayland_server *server) {
 		pid_t pid = fork();
 		if (pid < 0) {
 			wlr_log_errno(WLR_ERROR, "second fork failed");
+			(void) !write(p[1], "", 0);
 			_exit(EXIT_FAILURE);
 		} else if (pid == 0) {
```

Comment:
Should we `close(p[1])` here?

Reply:
Exiting the process will close the file descriptor.

Reply:
Right, I guess `exec_xwayland` will either exit or exec, so it doesn't matter.